docs/README.md

```diff
@@ -34,6 +34,55 @@ You'll need a `kubectl`-style config file to connect to your cluster.
 `$HOME/.kube/config`. You can specify an alternate kubeconfig connection file
 with `--kubeconfig`, or the env var `$KUBECONFIG`, for any command.
 
+## Kn Config
+
+To better customize Kn there is a small set of configuration parameters you are able to setup. In particular you can specify where your Kn plugins are located and how they are found. The Kn configuration file is meant to capture these limited configuration options. Let's explore this file, it's location, and the current options you are able to change with it.
+
+### Location
+
+By default the Kn config file is located in the user's home directory under `.kn` hidden subdirectory and is named `config.yaml`. So in `~/.kn/config.yaml`. It is not created for you by default. You can easily create one with:
```

Comment:
Can we use "`kn`" instead of "Kn" as this is the name how it should be typed in the CLI ? AFAIK nowhere we refer to kn as Kn so far.

Reply:
I use `kn` when referring to the binary and Kn when referring to the client. Are you suggesting to use `kn` everywhere?

Reply:
I'm fine also with just kn (instead of `kn`)

Reply:
OK changing to kn